app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java

```diff
@@ -327,6 +325,25 @@ private void postDescription(@NonNull String editToken) {
                     }));
         }
 
+        private Observable<MwPostResponse> getPostService(@NonNull String editToken, @Nullable String languageCode) {
+            if (invokeSource.name().contains(INVOKE_SOURCE_KEYWORD_CAPTION)) {
+                // TODO: update funnel
+                return ServiceFactory.get(wikiCommons).postLabelEdit(pageTitle.getWikiSite().languageCode(),
+                        pageTitle.getWikiSite().languageCode(), commonsDbName,
+                        pageTitle.getConvertedText(), editView.getDescription(),
+                        invokeSource == SUGGESTED_EDITS_ADD_DESC ? SuggestedEditsFunnel.SUGGESTED_EDITS_ADD_COMMENT
```

Comment:
These `invokeSource` checks should use `SUGGESTED_EDITS_ADD_CAPTION` and `SUGGESTED_EDITS_TRANSLATE_CAPTION`.

Reply:
Done